deployments/local-minio/configs/mis_config.yml

```diff
@@ -8,4 +8,4 @@ solr:
   host: solr
   port: 8983
 endpointUrl:
-  retrieve: http://cdr-ingest:9041/mis/product/
\ No newline at end of file
+  retrieve: http://multi-int-store:8080/mis/product/
```

Comment:
This should be updated to `http://localhost:9041/mis/product/`. This config value is only used to create the URIs that are returned by the query endpoint, and these URIs should be available to hit on a browser.

Reply:
I pushed a commit to address this.

Reply:
I was wrong when I said that is the only use. See https://github.com/connexta/multi-int-store/pull/119#discussion_r308919845.